src/bash/scripts/eval/set-prompt

```diff
@@ -26,6 +26,12 @@ __am_prompt_emit_prompt_arrow() {
     fi
 }
 
+__am_prompt_show_virtual_env() {
+  if [[ -n "$VIRTUAL_ENV" && -n "$DIRENV_DIR" ]]; then
+    echo "($(basename $VIRTUAL_ENV)) "
```

Comment:
spacing is different between this and the zsh version (this has a space after while zsh version has a space before). Not sure if that's intended.